.storybook/storybookStyles.scss

```diff
@@ -16,3 +16,9 @@
 [data-theme='medicare'] {
   @import '@cmsgov/ds-medicare-gov/src/styles/index';
 }
+
+article {
+  .preview {
+    border: 1px dashed orchid;
```

Comment:
I added this silly style to make the actual reset examples standout - we don't use dashed or purple borders so it made sense to me. This is temporary and will be removed upon approval of the reset.

Reply:
Regarding togglable styles, could this built-in feature do the same thing, Forrest?

<img width="718" alt="Screen Shot 2023-02-02 at 1 48 04 PM" src="https://user-images.githubusercontent.com/7595652/216457341-e3adc01c-a04e-4778-a5fb-3e46745957e8.png">

Or is the utility in the utility that is highlights which part is relevant?

Reply:
I didn't realize Storybook had this setting. It's a lot more granular, but appears to accomplish the same thing.